easybuild/easyblocks/s/superlu.py

```diff
@@ -123,22 +123,30 @@ def install_step(self):
         """
         super(EB_SuperLU, self).install_step()
 
-        expected_libpath = os.path.join(self.installdir, "lib", "libsuperlu.%s" % self.lib_ext)
-        actual_libpath   = os.path.join(self.installdir, "lib", "libsuperlu_%s.%s" % (self.cfg['version'], self.lib_ext))
+        self.libbits = ""
+        for libnames in ["lib", "lib64"]:
+            if os.path.exists(os.path.join(self.installdir, libnames)):
+                self.libbits = libnames
+
+        if not self.libbits:
+            raise EasyBuildError("No lib or lib64 directory exist in installdir '%s'" % self.installdir)
```

Comment:
@hajgato Using a `for` loop is a bit weird here imho, since there are only two options:

```python
self.libbits = 'lib'
if not os.path.exists(os.path.join(self.installdir, self.libbits)):
    self.libbits = 'lib64'

if not os.path.exists(os.path.join(self.installdir, self.libbits)):
    raise EasyBuildError("No lib or lib64 subdirectory exist in %s", self.installdir)

Reply:
@boegel Done.